config/sidebar.ts

```diff
@@ -20,9 +20,13 @@ function getTranslations(item: NavDict[number]): Record<string, string> | undefi
 
 type StarlightSidebarConfig = NonNullable<Parameters<typeof starlight>[0]['sidebar']>;
 
+type SidebarItem = Parameters<typeof starlight>[0]['sidebar']
+
 /** Generate a Starlight sidebar config object from our existing `nav.ts` files. */
 export function makeSidebar(): StarlightSidebarConfig {
-	let currentSubGroup: Extract<StarlightSidebarConfig[number], { items: any }>;
+	let currentSubGroup:
+		| Extract<StarlightSidebarConfig[number], { items: SidebarItem }>
+		| undefined;
```

Comment:
I think we can simplify this to this:

```suggestion
	let currentSubGroup: Extract<StarlightSidebarConfig[number], { items: StarlightSidebarConfig }>;
```